app/controllers/volume_mapping_controller.rb

```diff
@@ -0,0 +1,35 @@
+class VolumeMappingController < ApplicationController
+  include Mixins::GenericListMixin
+  include Mixins::GenericShowMixin
+  include Mixins::GenericSessionMixin
+  include Mixins::BreadcrumbsMixin
+  include Mixins::GenericFormMixin
+  include Mixins::GenericShowMixin
+  include Mixins::GenericButtonMixin
+
+  before_action :check_privileges
+  before_action :get_session_data
+  after_action :cleanup_action
+  after_action :set_session_data
+
+  def show
```

Comment:
this method should not be needed

Reply:
copy/pasted, assumed it's needed. removing.